main.js

```diff
@@ -0,0 +1,117 @@
+import articleService from './services/ArticleService.js';
+import productService from './services/ProductService.js';
+
+class Product {
+	name;
+	description;
+	price;
+	tags;
+	images;
+	favoriteCount;
+
+	constructor(name, description, price, tags, images, favoriteCount=0) {
+		this.name = name;
+		this.description = description;
+		this.price = price;
+		this.tags = tags;
+		this.images = images;
+		this.favoriteCount = favoriteCount;
+	}
+
+	favorite() {
+		this.favoriteCount++;
+	}
+}
+
+class ElectronicProduct extends Product {
+	manufacturer;
+
+	constructor(name, description, price, tags, images, favoriteCount=0, manufacturer) {
+		super(name, description, price, tags, images, favoriteCount=0);
```

Comment:
`favoriteCount=0`으로 생성자 인자를 넘기면 ElectronicProduct 생성자의 `favoriteCount` 파라미터값을 사용하지 않게 됩니다! 의도가 아니었다면 파라미터로 받은 값을 생성자로 넘겨주세요.


```suggestion
		super(name, description, price, tags, images, favoriteCount);
```